nexus/src/app/silo.rs

```diff
@@ -100,7 +100,14 @@ impl super::Nexus {
             .blueprint_target_get_current_full(opctx)
             .await
             .internal_context("loading target blueprint")?;
-        let target = target_blueprint.as_ref().map(|(_, blueprint)| blueprint);
+        let target = target_blueprint
```

Comment:
Maybe we want to have `blueprint_target_get_current_full()` return this 500 itself now that we expect all systems to have a target blueprint.

Reply:
This had more knock-on effects than I expected, but I think basically all positive: [78f9429](https://github.com/oxidecomputer/omicron/pull/5287/commits/78f942908af8d9535483f3e75466c24944a19de1)